sagan-common/src/test/java/sagan/projects/ProjectVersionOrderTests.java

```diff
@@ -37,13 +37,19 @@ public void getProjectReleases_ordersVersionsByNumber_milestones() throws Except
     @Test
     public void getProjectReleases_ordersVersionsByNumber_milestonesWithVersions() throws Exception {
         Project project = getProject("0.1 M1", "0.1", "0.1 M2");
-        assertThat(getProjectReleases(project), equalTo(asList("0.1 M2", "0.1 M1", "0.1")));
+        assertThat(getProjectReleases(project), equalTo(asList("0.1", "0.1 M2", "0.1 M1")));
+    }
+
+    @Test
+    public void getProjectReleases_ordersReleaseTrainsByName() throws Exception {
+        Project project = getProject("Brixton.SR7", "Camden.RELEASE", "Camden.SR5", "Camden.SR6", "Dalston.RELEASE");
+        assertThat(getProjectReleases(project), equalTo(asList("Dalston.RELEASE", "Camden.RELEASE", "Camden.SR6", "Camden.SR5", "Brixton.SR7")));
     }
```

Comment:
Curiously, Gradle itself believes the RELEASE comes after the SR because of the special meaning attached to the word RELEASE. `latest.*` constraints in Gradle would not properly select the SR.